docs/tutorials/cargo.md

```diff
@@ -0,0 +1,114 @@
+# Setting up your Cargo.toml file for ICU4X
```

Comment:
thought: increasingly of the opinion that we should be making an MdBook that contains an organized set of tutorials as well as deeper docs on e.g. stability policies

Reply:
Agreed; the docs folder gives us a lot of material we can use to start it